src/main/java/graphql/schema/idl/SchemaGenerator.java

```diff
@@ -183,8 +185,12 @@ public boolean isUseAppliedDirectivesOnly() {
             return useAppliedDirectivesOnly;
         }
 
+        public boolean getAddOnErrorDirective() {
+            return addOnErrorDirective;
+        }
```

Comment:
should be `isAddOnErrorDirective` in line with the other booleans in this Options class